ts_src/confidential.ts

```diff
@@ -57,16 +57,14 @@ export interface UnblindOutputResult {
   assetBlindingFactor: Buffer;
 }
 
-export function unblindOutput(
-  ephemeralPubkey: Buffer,
-  blindingPrivkey: Buffer,
+function unblindWithNonce(
```

Comment:
it should be called `unblindOutputWithNonce` and should accept a transaction output like unblindOutputWithKey does.